app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,54 @@
+class InvitesController < ApplicationController
+  skip_before_action :authenticate!, only: :accept
+
+  def create
+    begin
+      unless Invite.where('email = ? AND website_id = ? AND invited_by_id =?', invite_params[:email], current_website.id, current_user.id).blank?
+        raise "Duplicate invites"
+      end
+      invite = current_website.invites.new(invited_by_id: current_user.id, email: invite_params[:email])
+      UserMailer.member_invitation(invite.id).deliver_later if invite.save
+      redirect_to new_invite_url, notice: 'Email sent'
+
+    rescue Exception => e
+      redirect_to(:action => 'new', notice: e.message)
+    end
+  end
```

Comment:
Use included exception rescue here into the method that means you should do this:
```
def
...
rescue => e
...
end
```

Without begin and without double ends.

There is also another way of doing this without rescue and that is by doing the redirect above with one check. Like so:
```
redirect_to(:new, notice: 'User already invited') && return unless current_website.invites.where(email: invite_params[:email]).blank?
```

If you add the and return at the end then it's going to stop execution.